docs/customizing.md

```diff
@@ -164,6 +164,8 @@ be created successfully.
 
 Check the monitoring namespace (or the namespace you have specific in `namespace: `) and make sure the pods are running. Prometheus and Grafana should be up and running soon.
 
+If you want to increase retention time make sure you will add retention: 1y in `prometheus-prometheus.yaml `
```

Comment:
I don't think it's the right place otherwise we would have to enumerate all possible settings. If your workflow is to update manually the YAML manifests generated by kube-prometheus then the right source of information is https://prometheus-operator.dev/docs/operator/api/#monitoring.coreos.com/v1.PrometheusSpec (it has a description of the CRD fields).
Having said there could be a parameter in https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/prometheus.libsonnet to define the retention value?